app/assets/javascripts/discourse/tests/acceptance/sidebar-user-messages-section-test.js

```diff
@@ -376,6 +376,10 @@ acceptance(
         "service:pm-topic-tracking-state"
       );
 
+      await click(
+        ".sidebar-section-messages .sidebar-section-link-group-messages-inbox.group1"
+      );
+
```

Comment:
Moving this above the `publishToMessageBus` calls since we now have to visit the personal messages page to be subscribed to the messageBus channels.